src/app/organization/employees/employees.component.html

```diff
@@ -0,0 +1,61 @@
+<div class="container m-b-20" fxLayout="row" fxLayoutAlign="end" fxLayoutGap="20px">
+
+  <button mat-raised-button color="primary">
+    <fa-icon icon="upload"></fa-icon>&nbsp;&nbsp;
+    Import Employees
+  </button>
+
+  <button mat-raised-button color="primary">
+    <fa-icon icon="plus"></fa-icon>&nbsp;&nbsp;
+    Create Employees
+  </button>
+
+</div>
+
+<div class="container">
+
+  <div fxLayout="row" fxLayoutGap="20px">
+    <mat-form-field fxFlex>
+      <mat-label>Filter</mat-label>
+      <input matInput (keyup)="applyFilter($event.target.value)">
+    </mat-form-field>
+  </div>
+
+  <div class="mat-elevation-z8">
+
+    <table mat-table [dataSource]="dataSource" matSort>
+
+      <ng-container matColumnDef="displayName">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Name </th>
+        <td mat-cell *matCellDef="let employee"> {{ employee.displayName }} </td>
+      </ng-container>
+
+      <ng-container matColumnDef="isLoanOfficer">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Is Loan Officer </th>
+        <td mat-cell *matCellDef="let employee"> {{ employee.isLoanOfficer }} </td>
```

Comment:
Let's update this with icons similar to what's currently implemented in the web app components like System => Codes etc.

Reply:
@punwai For loan officer as I specified take System => Codes as an example where we have icons `check-circle` and `times-circle`.